web-client/public/js/grnstate.js

```diff
@@ -1,3 +1,46 @@
+let currentNetwork = null;
+
+const annotateLinks = network => {
+    // TODO This duplicates logic that is done on the server side for an .xlsx spreadsheet.
+    //      Think of a way to consolidate it. Having discovered this, it seems like this should
+    //      be done on the client side because it rearranges data redundantly, for ease of display.
```

Comment:
This `TODO` comment was pre-existing and still looks like it is worth pursuing eventually.